addons/web/models/models.py

```diff
@@ -84,6 +84,14 @@ def _format_web_search_read_results(self, domain, records, offset=0, limit=None,
             'records': records,
         }
 
+    def web_save(self, vals, specification: Dict[str, Dict]) -> List[Dict]:
+        if (self.id):
```

Comment:
```suggestion
        if len(self):
```
At least remove parenthesis